simulation-scripts/scenario/rbac-shoot-and-scoot/challenge.txt

```diff
@@ -1,9 +1,9 @@
-Scenario: A developer has informed you, they are able to make Kubernetes API calls from an Ubuntu pod although it only belongs to the default service account.
+Scenario: A developer has informed you, they are able to make Kubernetes API calls from an Ubuntu pod to get a secret although its service account token is unmounted.
 
 Starting Point: ubuntu pod in the rbac-shoot-and-scoot namespace
 
 Difficulty: Easy
 
-Task 1: Can you replicate this issue.
+Task 1: Can you replicate this issue by finding the secret value in the rbac-shoot-and-scoot namespace.
```

Comment:
"namespace?"

Reply:
The secret value is in a secret object in the namespace

Reply:
Apologies, I was trying to highlight the lack of the question mark.

Reply:
Lol, will fix